docs/pages/router/basics/common-navigation-patterns.mdx

```diff
@@ -136,68 +137,89 @@ When you're already focused on a tab and navigating to a user, you will stay in
 
 ## Authenticated users only: protected routes
 
-If you have a set of routes that should only be accessible to authenticated users, you can embed those routes in a group whose layout redirects users to a login page if they are not authenticated.
+For mobile apps requiring authentication, you will likely have a set of routes that should only be accessible to authenticated users.
 
-Here's what that looks like:
+Say we want to have a bottom tabs layout, a sign in page, a create account page, and a modal that should only be visible to authenticated users:
 
 <FileTree
   files={[
-    ['app/_layout.tsx'],
-    ['app/login.tsx', 'routes users back to /(logged-in) after authentication'],
-    ['app/(logged-in)/\_layout.tsx', 'includes redirect for unauthenticated users'],
-    ['app/(logged-in)/(tabs)/\_layout.tsx'],
-    ['app/(logged-in)/(tabs)/index.tsx', 'default route for app'],
-    ['app/(logged-in)/(tabs)/feed.tsx'],
-    ['app/(logged-in)/modal.tsx'],
+    ['app/_layout.tsx', 'Root layout'],
+    ['app/(tabs)/_layout.tsx'],
+    [
+      'app/(tabs)/index.tsx',
+      <span>
+        Protected <Lock01Icon className="mb-1 inline" />
+      </span>,
+    ],
+    [
+      'app/(tabs)/settings.tsx',
+      <span>
+        Protected <Lock01Icon className="mb-1 inline" />
+      </span>,
+    ],
+    ['app/sign-in.tsx'],
+    ['app/create-account.tsx'],
+    [
+      'app/modal.tsx',
+      <span>
+        Protected <Lock01Icon className="mb-1 inline" />
+      </span>,
+    ],
   ]}
 />
 
-By default, the app will go to the nearest index, **app/(logged-in)/(tabs)/index.tsx**. However, a route group's layout file will be rendered before the enclosed route is. So, if the user is not authenticated, at this point, you can redirect the user to the login page:
+When your app is first launched, router will try to open the root index, **app/(tabs)/index.tsx**. If we wrap this screen in a `Stack.Protected` with the `guard={false}`, the screen will become inaccessible and the next available screen will be opened instead. In our example, the `sign-in` screen will be opened, since it is the next available route.
```

Comment:
```suggestion
When your app is first launched, the router will try to open the root index, **app/(tabs)/index.tsx**. If you wrap this screen in a `Stack.Protected` with the `guard={false}`, the screen will become inaccessible and the next available screen will be opened instead. In this example, the `sign-in` screen will be opened, since it is the next available route.
```

Let's avoid using first-person voice as per the writing style guide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#write-in-the-second-person